third_party/fiat/p384_64.h

```diff
@@ -26,15 +26,6 @@ __extension__ typedef __uint128_t fiat_p384_uint128;
 #error "This code only works on a two's complement system"
 #endif
 
-#if !defined(FIAT_P384_NO_ASM) && (defined(__GNUC__) || defined(__clang__))
```

Comment:
These definitions were not used anyway as stated on l.133, right?

Reply:
correct